### PR TITLE
Add first version of German i18n guide

### PR DESCRIPTION
**Pull request by hippotastic**
#### What kind of changes does this PR include?

- Something else!

#### Description

- Adds first version of our German i18n guide for translators. Only visible in the GitHub repo.

### CODE REVIEW

**Comment by nbyx** [comment]
LGTM! :)

File: src/i18n/de/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+| frontmatter                 | das Frontmatter                      | Wird nicht übersetzt (gängiger Domänenbegriff).
+| frontmatter prop(erty)      | die Frontmatter-Eigenschaft          |
+| frontmatter value           | der Frontmatter-Wert                 |
+| page                        | die Seite                            | Eine einzelne (HTML-)Seite. Wir vermeiden die längere Form "Webseite", um Verwechslungen mit "Website" zu vermeiden.
+| repository                  | das Repository                       | Wird nicht übersetzt (gängiger Domänenbegriff).
+| script                      | das Skript                           |
+| site, website               | die Website                          | Eine vollständige Webpräsenz unter einer gemeinsamen Domain, bestehend aus beliebig vielen (HTML-)Seiten.
+| template                    | die Vorlage                          |
+| web                         | das Internet                         |
+
```
Comment (tobiasschmidt89) [inline src/i18n/de/README.md:39]: I would add: UI, Partial Hydration and Isles.

Also configuration info is needed often so we should norm how we name the astro.config.mjs fields. E.g. Parameter, Felder, Schlüssel or Werte...

**Review by tobiasschmidt89 — COMMENTED** [review]
See comment for additional suggestions for terms.

**Review by delucis — APPROVED** [review]
LGTM! Thanks for putting this together for us @hippotastic 💜

File: src/i18n/de/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+## Übersetzungs-Glossar
+
+| Originalbegriff             | Übersetzung                          | Anmerkungen
+|:----------------------------|:-------------------------------------|:------------
+| branch                      | der Branch                           | Wird nicht übersetzt (gängiger Domänenbegriff).
+| build                       | Erzeugung / Erstellung               | Wird aus Konsistenzgründen zu bestehenden Übersetzungen vorerst so übersetzt. Eventuell werden wir "Build" aber zukünftig als gängigen Domänenbegriff einstufen und nicht übersetzen.
+| build process               | Erzeugungs- / Erstellungsvorgang     | s.o.
+| build time                  | Erzeugungs- / Erstellungszeitpunkt   | s.o.
+| to build                    | erzeugen                             | Da die Alternativen "bauen" oder gar "builden" merkwürdig klingen, wird diese Übersetzung vermutlich so bleiben.
+| command line                | die Kommandozeile                    |
```
Comment (space-otter) [inline src/i18n/de/README.md:24]: Commit = Commit: Semantisch gibt es keinen deutschen treffenden Begriff, somit würde ich ebenfalls Commit beibehalten. Weiter könnte man generell Begrifflichkeiten aus dem Git-Ökosystem als etablierte Neuologismen (etablierter Neologismus, Kotradiktion in sich selbst lol) etablieren.

Comment (hippotastic) [reply src/i18n/de/README.md:24]: Good idea! Feel free to make a new PR adding this as this PR has already been merged. :)

**Pull request merged**